File: src/main/java/no/nav/service/pensjon/regler/to/SamletEktefellepensjonResponse.java
package no.nav.service.pensjon.regler.to;


import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.beregning2011.AbstraktBeregningsResultat;
import no.nav.domain.pensjon.regler.beregning2011.BeregningsResultatAfpPrivat;

public class SamletEktefellepensjonResponse extends ServiceResponse  {
	private AbstraktBeregningsResultat soker1Beregningsresultat;
	private AbstraktBeregningsResultat soker2Beregningsresultat;
	private Beregning soker1Beregning;
	private Beregning soker2Beregning;
	private BeregningsResultatAfpPrivat soker1BeregningsResultatAfpPrivat;
	private BeregningsResultatAfpPrivat soker2BeregningsResultatAfpPrivat;
	private Pakkseddel pakkseddel;


}

File: src/main/kotlin/no/nav/pensjon/regler/to/SamletEktefellepensjonResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.beregning2011.AbstraktBeregningsResultat
import no.nav.pensjon.regler.domain.beregning2011.BeregningsResultatAfpPrivat


class SamletEktefellepensjonResponse : ServiceResponse() {
    //return soker1Beregningsresultat;
    var soker1Beregningsresultat: AbstraktBeregningsResultat? = null

    //return soker2Beregningsresultat;
    var soker2Beregningsresultat: AbstraktBeregningsResultat? = null
    var soker1Beregning: Beregning? = null
    var soker2Beregning: Beregning? = null
    var soker1BeregningsresultatAfpPrivat: BeregningsResultatAfpPrivat? = null
    var soker2BeregningsresultatAfpPrivat: BeregningsResultatAfpPrivat? = null
}